.travis.yml

```diff
@@ -2,12 +2,17 @@ sudo: false
 
 language: php
 
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
I would only cache `$HOME/.composer/cache/files` here. Metadata are likely to change far more often.